install/kubernetes/addons/grafana.yaml

```diff
@@ -17,12 +17,12 @@ kind: Deployment
 metadata:
   name: grafana
   namespace: istio-system
+  annotations:
+    sidecar.istio.io/inject: "false"
 spec:
   replicas: 1
   template:
     metadata:
-      annotations:
-        alpha.istio.io/sidecar: ignore
```

Comment:
This is generated, can you remove from grafana.yaml.tmpl and re-run updateVersion.sh

Reply:
That should be the case already. Lines 20-21 show the result of the new annotation from the template.

Reply:
Nvm, it's good, somehow I had missed the .tmpl files.